### PR TITLE
feat: update to rust 1.9.0

### PR DESCRIPTION
**Pull request by emig**
changes to make the project build with rust 1.9.0
rust-fm applied

### CODE REVIEW

File: src/c51.rs [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 
 impl Train for C51 {
-  fn body(&self) -> &'static [&'static str] {
-    &C51BODY
-  }
+    fn body(&self) -> &'static [&'static str] {
+        &C51BODY
+    }
 
-  fn wheelset(&self, x: uint) -> &'static [&'static str] {
```
Comment (skade) [inline src/c51.rs]: The correct translation for uint is usize. No need for the cast later.

File: src/d51.rs [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
 impl Train for SL {
-  fn body(&self) -> &'static [&'static str] {
-    &D51BODY
-  }
+    fn body(&self) -> &'static [&'static str] {
+        &D51BODY
+    }
 
-  fn wheelset(&self, x: uint) -> &'static [&'static str] {
-    &D51WHL[x % 6]
```
Comment (skade) [inline src/d51.rs]: Same here

File: src/logo.rs [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
 impl Train for Logo {
-  fn body(&self) -> &'static [&'static str] {
-    &LOGO
-  }
+    fn body(&self) -> &'static [&'static str] {
+        &LOGO
+    }
 
-  fn wheelset(&self, x: uint) -> &'static [&'static str] {
-    &LWHL[x % 6]
```
Comment (skade) [inline src/logo.rs]: Same.

File: src/logo.rs [hunk 4]
```diff
@@ -1,27 +1,27 @@
 use super::Train;
 use super::data::{LOGO, LWHL, LCOAL, LCAR};
 
-#[derive(Copy)]
+#[derive(Copy, Clone)]
```
Comment (skade) [inline src/logo.rs]: I'm not sure why clone is necessary here.

Comment (emig) [inline src/logo.rs]: Copy requires Clone

Comment (skade) [inline src/logo.rs]: Oh, you are right :).

**Pull request merged**